management/templates/system-backup.html

```diff
@@ -374,4 +377,14 @@ <h3>Available backups</h3>
     }
 };
   
+// hide Copy button if not in a modern clipboard-supporting environment
```

Comment:
```suggestion
// Hide Copy button if not in a modern clipboard-supporting environment.
// Using document API because jQuery is not necessarily available in this script scope.
```